nextcord/state.py

```diff
@@ -2259,6 +2264,57 @@ def parse_guild_scheduled_event_user_remove(self, data) -> None:
                 data["guild_id"],
             )
 
+    def add_automod_rule(self, data: AutoModerationRulePayload) -> AutoModerationRule:
+        rule = AutoModerationRule(state=self, guild=self._get_guild(int(data["guild_id"])), data=data)  # type: ignore -- automod rules can't be created in non-guilds
+        self._automod_rules[int(data["id"])] = rule
+        return rule
+
+    def delete_automod_rule(self, id: int) -> None:
+        self._automod_rules.pop(id)
```

Comment:
```suggestion
        self._automod_rules.pop(id, None)
```
silently ignores popping `ValueError`s

Reply:
done